regimes/gr/tax_categories.go

```diff
@@ -0,0 +1,99 @@
+package gr
+
+import (
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/tax"
+)
+
+var taxCategories = []*tax.Category{
+	//
+	// VAT
+	//
+	{
+		Code: tax.CategoryVAT,
+		Name: i18n.String{
+			i18n.EN: "VAT",
+			i18n.EL: "ΦΠΑ",
+		},
+		Title: i18n.String{
+			i18n.EN: "Value Added Tax",
+			i18n.EL: "Φόρος προστιθέμενης αξίας",
+		},
+		Sources: []*tax.Source{
+			{
+				Title: i18n.String{
+					i18n.EN: "Value Added Tax/Goods and Services Tax (VAT/GST) (1976-2023)",
+					i18n.EL: "Φόρος Προστιθέμενης Αξίας/Φόρος Αγαθών και Υπηρεσιών (ΦΠΑ/GST) (1976-2023)",
+				},
+				URL: "https://www.oecd.org/tax/tax-policy/tax-database/",
+			},
+		},
+		Retained: false,
+		Extensions: []cbc.Key{
+			ExtKeyIAPRVATCat,
+		},
+		Rates: []*tax.Rate{
+			{
+				Key: tax.RateStandard,
+				Name: i18n.String{
+					i18n.EN: "Standard rate",
+					i18n.EL: "Κανονικός συντελεστής",
+				},
+				Values: []*tax.RateValue{
+					{
+						Percent: num.MakePercentage(24, 2),
+					},
+				},
+				Map: cbc.CodeMap{
+					KeyIAPRVATCategory: "1",
+				},
+			},
+			{
+				Key: tax.RateReduced,
+				Name: i18n.String{
+					i18n.EN: "Reduced rate",
+					i18n.EL: "Μειωμένος συντελεστής",
+				},
+				Values: []*tax.RateValue{
+					{
+						Percent: num.MakePercentage(13, 2),
+					},
+				},
+				Map: cbc.CodeMap{
+					KeyIAPRVATCategory: "2",
+				},
+			},
+			{
+				Key: tax.RateSuperReduced,
+				Name: i18n.String{
+					i18n.EN: "Super-Reduced Rate",
+					i18n.EL: "Υπερμειωμένος συντελεστής",
+				},
+				Values: []*tax.RateValue{
+					{
+						Percent: num.MakePercentage(6, 2),
+					},
+				},
+				Map: cbc.CodeMap{
+					KeyIAPRVATCategory: "3",
+				},
+			},
```

Comment:
We lost the island rates here... I do think they can still be supported using the `standard+island` (`tax.RateStandard.With(RateIsland)`) format in the Key.

Reply:
I thought they weren't essential at this stage, and I wasn't sure about the `+` idea. I'll add them!